### DISCUSSION TITLE
Stackers Badges

### DISCUSSION THREAD

**Discussion by Darth-Coin** [stackernews/stacker.news#1870] [Ideas]
**Is your feature request related to a problem? Please describe.**
No, is more like a experience improvement and some kind of WoT...

**Describe the solution you'd like**
Stackers to be able to gift and "mark" another stacker with a personalized badge, in form of a small image or icon.
Something like https://nostrbadges.com/.
But hey... these badges are NOT free and not cheap! But are public visible (somehow, in the profile or everywhere).

Could be:
- personal individual badges, each stacker can create a set of badges, sell them or offer them.
- general SN badges, that can be bought from SN saloon and then rewarded to others.

Also these badges can be traded (ofc with cowboy credits) in the saloon (only) and also if somebody want to remove a badge "marked" upon his profile, must pay for it.

I think this can improve a lot the WoT and identify the scammers and assmilkers.

**Comment by huumn** [comment]
Sounds fun!
